frontend/src/metabase-lib/queries/utils/virtual-table.ts

```diff
@@ -23,6 +23,8 @@ export function createVirtualTable({
   db,
   ...rest
 }: VirtualTableProps): Table {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
It's incorrect that virtual tables created without ids (and other fields). Also to be fixed separately